### ISSUE TITLE
PHP warning: trim() expects parameter 1 to be string, array given

### ISSUE THREAD

**Issue by Ibochkarev** [modxcms/revolution#15061]
### Summary
There are many repetitive PHP warnings in the error log

### Step to reproduce
Go to error log

### Observed behavior
`[2020-04-23 08:55:53] (ERROR @ /var/www/****/htdocs/core/model/modx/modtemplatevar.class.php : 749) PHP warning: trim() expects parameter 1 to be string, array given
[2020-04-23 09:05:53] (ERROR @ /var/www/****/htdocs/core/model/modx/modtemplatevar.class.php : 749) PHP warning: trim() expects parameter 1 to be string, array given
[2020-04-23 09:15:53] (ERROR @ /var/www/****/htdocs/core/model/modx/modtemplatevar.class.php : 749) PHP warning: trim() expects parameter 1 to be string, array given
[2020-04-23 09:25:53] (ERROR @ /var/www/****/htdocs/core/model/modx/modtemplatevar.class.php : 749) PHP warning: trim() expects parameter 1 to be string, array given
[2020-04-23 09:28:13] (ERROR @ /var/www/****/htdocs/core/model/modx/modtemplatevar.class.php : 749) PHP warning: trim() expects parameter 1 to be string, array given
[2020-04-23 09:28:14] (ERROR @ /var/www/****/htdocs/core/model/modx/modtemplatevar.class.php : 749) PHP warning: trim() expects parameter 1 to be string, array given`

**Possible Solution**:
In file _core/model/modx/modtemplatevar.class.php_
in 748 line add:
``` php
if (is_string($value)) {
    $nvalue = trim($value);
}
```

```php
public function getBindingDataFromValue($value) {
        if (is_string($value)) {
            $nvalue = trim($value);
        }
        $cmd = false;
        $param = '';
        if (substr($nvalue,0,1) == '@') {
            list($cmd,$param) = $this->parseBinding($nvalue);
            $cmd = trim($cmd);
        }
        return array('cmd' => $cmd,'param' => $param);
    }
```

### Environment
MODX Revolution 2.7.1-pl (traditional)
PHP Version 7.3.16-1
mysqlnd 5.0.12-dev

**Comment by JoshuaLuckers** [comment]
Could you share the info needed to reproduce this?

**Comment by Ibochkarev** [comment]
> Could you share the info needed to reproduce this?

Unfortunately I can’t understand the reason for the occurrence, but after the fix, the notifications disappeared

**Comment by JoshuaLuckers** [comment]
Can you share the template variables you are using (you can Slack me if it's private)?

**Comment by garryn** [comment]
Is this still an issue?  (@JoshuaLuckers could you reproduce it) ... just wondering :)

**Comment by JoshuaLuckers** [comment]
I have not been able to reproduce it.

**Comment by JoshuaLuckers** [comment]
@Ibochkarev it would be really great if we could reproduce this issue. Do you know what template variable's you where/are using?

**Comment by Ibochkarev** [comment]
Fix #15624
